src/dto/parameters/filter-pagination.dto.ts

```diff
@@ -0,0 +1,34 @@
+import { Transform, TransformFnParams } from 'class-transformer';
+import {
+  IsDate,
+  IsInt,
+  IsOptional,
+  IsPositive,
+  Length,
+  MinDate,
+} from 'class-validator';
+import { MIN_DATE_VALUE, TASK_MIN_LENGTH } from 'common';
+
+export class FilterPaginationDto {
+  @IsOptional()
+  @IsInt()
+  @IsPositive()
+  @Transform(({ value }: TransformFnParams) => parseInt(value, 10))
```

Comment:
do you know what is wrong here? I guess yes

Reply:
it removed

Reply:
fix it